core/src/main/java/io/aiven/klaw/controller/KafkaConnectController.java

```diff
@@ -75,6 +83,7 @@ public ResponseEntity<List<KafkaConnectorRequestsResponseModel>> getCreatedConne
         HttpStatus.OK);
   }
 
+  @PermissionAllowed(permissionAllowed = {PermissionType.REQUEST_DELETE_CONNECTORS})
```

Comment:
Probably the implemented code is not correct, which is looking for REQUEST_CREATE_CONNECTORS ?

Reply:
Here do we need a permission, or rather check if the user owns that request ?